src/log/access_log.cc

```diff
@@ -474,10 +477,16 @@ fvdbCount(hash_table * hash, const char *key)
     ++ fv->n;
 }
 
+static void
+fvdbCount(fvdb_table_t &table, const SBuf& key)
```

Comment:
> It can be removed only after forw_table has been refactored, too.

AFAICT, forw_table-related code does not use this function. This function was added by this PR. Removing this function will not affect forw_table-related code. Am I missing something?

Reply:
Looks to me like you have it right @rousskov . The fvdbCountVia can just use ++table[X] and this one-liner goes away again.

Reply:
Removed. Things will furhter simplify when we get rid of forw_table as well